### PR TITLE
Updated parameter name and summary

### PR DESCRIPTION
**Pull request by shaggygi**
Updated parameters to be consistent and added summary.

### CODE REVIEW

File: src/devices/Mcp23xxx/samples/Mcp23xxx.Samples.csproj [hunk 1]
```diff
@@ -6,13 +6,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <!--
-        Temporarily making a project reference until the daily has IGpioController
-        <PackageReference Include="System.Device.Gpio" Version="0.1.0-prerelease*" /> 
-    -->
```
Comment (joperezr) [inline src/devices/Mcp23xxx/samples/Mcp23xxx.Samples.csproj]: you can remove the above PackageReference to System.Device.Gpio package as well, as the project reference should bring in the gpio library as a transitive reference.

File: src/devices/Mcp23xxx/tests/Mcp23xxx.Tests.csproj [hunk 2]
```diff
@@ -2,25 +2,15 @@
...
     <PackageReference Include="xunit.runner.visualstudio" Version="2.4.1" />
-    <!--
-      Temporarily making a project reference until the daily has IGpioController
-      <PackageReference Include="System.Device.Gpio" Version="0.1.0-prerelease*" /> 
-    -->
-    <ProjectReference Include="..\..\..\System.Device.Gpio\System.Device.Gpio.csproj" >
-      <AdditionalProperties>Configuration=Linux-Debug;RuntimeIdentifier=linux</AdditionalProperties>
-    </ProjectReference>
+    <PackageReference Include="System.Device.Gpio" Version="0.1.0-prerelease*" />
+    <ProjectReference Include="$(MSBuildThisFileDirectory)\..\Mcp23xxx.csproj" />
```
Comment (joperezr) [inline src/devices/Mcp23xxx/tests/Mcp23xxx.Tests.csproj]: Same here, you should be able to remove the Gpio package reference

**Comment by joperezr** [comment]
@shaggygi  Changes look good to me, but I'm not sure why the MCP tests are failing to build, I don't see why your changes are causing it. Can you repro this locally?

**Comment by shaggygi** [comment]
@joperezr i didn't seem to be failing on my end.  That's why there were a few comments updating the packages.  I'll try and review/update again tonight.

**Comment by joperezr** [comment]
I'll try to pull your changes and repro the issue as well.

**Comment by shaggygi** [comment]
@joperezr tried again with no luck.  I did have to update to match the changes for ReadOnlySpan after removing the packages.  Builds fine on my end 🤷‍♂️

**Comment by shaggygi** [comment]
@krwq @joperezr @JeremyKuhne i'm completely confused on this error.  it has something to do with the Mcp23xxx test project.  I tried updating to other tests projects and removing/adding with no luck.  The builds pass when removing from repo.  It builds and all tests pass on my local machine.  I give up.  Going to drink heavily now.  🍻

**Comment by joperezr** [comment]
I’ll take a look tomorrow morning.

File: src/devices/Mcp23xxx/tests/Mcp23xxxTest.cs [hunk 3]
```diff
@@ -120,8 +123,9 @@ public Mcp23xxxChipMock(int ports, bool isSpi)
             public Span<byte> Registers => _registers;
 
             // Can't coalesce here https://github.com/dotnet/roslyn/issues/29927
-            public ReadOnlySpan<byte> LastReadBuffer => _lastReadBuffer == null ? ReadOnlySpan<byte>.Empty : _lastReadBuffer;
-            public ReadOnlySpan<byte> LastWriteBuffer => _lastWriteBuffer == null ? ReadOnlySpan<byte>.Empty : _lastWriteBuffer;
+            public ReadOnlySpan<byte> LastReadBuffer => _lastReadBuffer ?? ReadOnlySpan<byte>.Empty;
```
Comment (joperezr) [inline src/devices/Mcp23xxx/tests/Mcp23xxxTest.cs]: This is the issue that is causing the build break. As you see in the comment above, there is an existing compiler issue causing the 'Failed to emit module' error when you try to Coalesce with the ?? operator. I tried building with all your changes + reverting these two lines and building works fine for me now.

Comment (shaggygi) [reply src/devices/Mcp23xxx/tests/Mcp23xxxTest.cs]: Ah, good catch.  And I remember seeing that when changing.  Little confused on why it worked on my end, though.  Anyways, builds look like they're working now.  Thx for finding this.

**Comment by shaggygi** [comment]
@krwq you think you can squeeze some time in to review this?  I wanted to finalize this before moving on to my assigned issues (which are somewhat related to the changes).  Thx

**Comment by joshfree** [comment]
Thanks @shaggygi for your contributions 😄

**Pull request merged**